app/Http/Controllers/Api/SettingsController.php

```diff
@@ -150,8 +150,11 @@ public function ajaxTestEmail() : JsonResponse
         if (!config('app.lock_passwords')) {
             try {
                 Notification::send(Setting::first(), new MailTest());
+                Log::debug('Attempting to sending to '.config('mail.reply_to.address'));
```

Comment:
This one either - we don't need to know about it is it didn't trip the catch exception handler.

Reply:
It really isn't. If no exception is being thrown, there's no information to be gained from that. I'll allow it in this case, but we don't normally like to junk up the logs too much. Idk log files are not writeable for some reason (which happens quite a lot with our users) it can break things in very unexpected ways.

I promise you don't need to explain how mail servers or http status codes work to us.

Reply:
Ok got it

Reply:
I will try to do best for track this king of issues with mail, i used mailtrap in testing env and mailchip in production, thanks for help us.